docs/api.rst

```diff
@@ -119,27 +114,47 @@ Inference
 
 .. currentmodule:: pyhf.infer
 
+
+Test Statistics
+~~~~~~~~~~~~~~~
+
 .. autosummary::
    :toctree: _generated/
-   :template: modifierclass.rst
+   :nosignatures:
 
    test_statistics.q0
    test_statistics.qmu
    test_statistics.qmu_tilde
    test_statistics.tmu
    test_statistics.tmu_tilde
-   mle.twice_nll
-   mle.fit
-   mle.fixed_poi_fit
-   hypotest
-   intervals.upperlimit
+   utils.get_test_stat
```

Comment:
I get the idea of trying to group the `utils` bits together with their area of use, but then it seems a bit confusing to have the API "path" be different and still be under the same heading. What are your thoughts on making a small "Utils" section that would go under "Fits and Tests" that would just have `utils.get_test_stat` and `utils.create_calculator`?

Reply:
Well, then we'd do the same thing for "Fits and Tests" since those are different API paths too... All of this is under `pyhf.infer` and I'm not sure we need to split by anything further in the API for subsection.

Reply:
fair.